c/test/CMakeLists.txt

```diff
@@ -25,29 +25,31 @@ if(APPLE)
   list(APPEND TEST_INCLUDES "-L/usr/local/lib")
 endif()
 
-swift_add_test(test-libsbp-legacy
-  UNIT_TEST
-  SRCS
-    check_main_legacy.c
-    check_edc.c
-    check_sbp.c
-    check_bitfield_macros.c
-    ${generated_legacy_c_sources}
-  INCLUDE
-    ${TEST_INCLUDES}
-  LINK
-    ${TEST_LIBS}
+if (NOT CMAKE_C_COMPILER_ID STREQUAL "MSVC")
```

Comment:
MSVC doesn't support any zero array length and a number of other stuff, hence the removal of this and the legacy C++ library

Reply:
Is there a compiler flag for that anywhere? In `common.h` we have some #define for MSVC which suggests there has been at lest historical use of the legacy API

Reply:
Good point, looks like there was evidence to support MSVC - https://github.com/swift-nav/libsbp/pull/568 - but there doesn't seem to be any build system enforcing it. Rerunning the build it was evident that the problem was in `msg_fileio_write_req_t` which had 2 zero array length member variables, which is a problem and something that still surprises me that GCC and Clang are okay with.